ios/src/main/java/org/ergoplatform/ios/api/IosEncryptionManager.kt

```diff
@@ -0,0 +1,47 @@
+package org.ergoplatform.ios.api
+
+import org.ergoplatform.api.AesEncryptionManager
+import org.ergoplatform.utils.LogUtils
+import org.robovm.apple.localauthentication.LAContext
+import kotlin.random.Random
+
+object IosEncryptionManager {
+    private const val KEYCHAIN_SERVICE = "org.ergoplatform.ios"
+    private const val KEYCHAIN_ACCOUNT = "ergoWalletKey"
+    private const val KEY_LENGTH = 28
+
+    fun encryptDataWithKeychain(data: ByteArray, context: LAContext): ByteArray {
+        val key = loadOrGenerateKey(context)
+        return AesEncryptionManager.encryptData(key, data)
+    }
+
+    fun decryptDataWithKeychain(encryptedData: ByteArray, context: LAContext): ByteArray {
+        val key = loadOrGenerateKey(context)
+        return AesEncryptionManager.decryptData(key, encryptedData)
+    }
+
+    private fun getRandomKey(): String {
+        val chars = ('a'..'Z') + ('A'..'Z') + ('0'..'9')
+        val rnd = Random(System.nanoTime())
+        return List(KEY_LENGTH) { chars.random(rnd) }.joinToString("")
```

Comment:
I'm not sure this is secure random generator. Please look alternative from one of the security packages in JDK or bouncycastle.

Reply:
Changed to `SecureRandom()`